README.md

```diff
@@ -60,6 +60,41 @@ $$
 function_info = client.create_function(sql_function_body=sql_body)
 ```
 
+#### Create a UC python function
+
+The client offers an API that simplifies the process of directly creating a python function to be registered as a UC function.
+
+To get the optimal results for GenAI tool definitions, it is highly recommended to:
+
+- Provide a full Python docstring in Google docstring format to your function definition. This API extracts the information about arguments so that GenAI tools
+will have additional context about the tool's capabilities.
+- Ensure that the description of the function is as descriptive as possible to prevent a tool use call that infers capabilities of the tool purely based on the
+function's name.
+- Use a clear and descriptive naming convention for the function to aid in identification within Unity Catalog as well as for GenAI systems to understand
+additional context about a configured tool within an agentic application.
+- Use type hints. Failure to provide them will cause an exception to be raised.
```

Comment:
What types are supported? Only primitives such as `int`?

Reply:
I'll add the full list of what's permitted and what is guaranteed not to work - basically if it's in the core lib type declarations or in the typing module, we'll pick it up, but the mapping to Spark SQL types is restricted to the enumeration that I created. I'll add a section on that in the doc string and the README :)